pages/about.html

```diff
@@ -5,8 +5,24 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="styles.css" rel="stylesheet">
 </head>
 <body>
-  
+  <main class = "container">
+  <header>
```

Comment:
The header tag generally wouldn't go within the main tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

We'll often use them as
```html
  <header>Some header</header>
  <main>Main content</main>
  <footer>Some footer</footer>
```